### ISSUE TITLE
Doc2Vec.py ZeroDivisionError: float division by zero

### ISSUE THREAD

**Issue by kikohs** [piskvorky/gensim#529]
Hello,
I ran into this bug while launching multiple instance of doc2vec in parallel using `scikit-learn cross-validation`. I am basically trying to find the best parameter for doc2vec using sklearn `RandomGridSearchCV`

In doc2vec.py 

    self.scan_vocab(sentences, trim_rule=trim_rule)  # initial survey
	 for document_no, document in enumerate(documents):
	    621             if document_no % progress_per == 0:
	--> 622                 interval_rate = (total_words - interval_count) / (default_timer() - interval_start)
	    623                 logger.info("PROGRESS: at example #%i, processed %i words (%i/s), %i word types, %i tags",
	    624                             document_no, total_words, interval_rate, len(vocab), len(self.docvecs))
	    625                 interval_start = default_timer()
	    626                 interval_count = total_words

	ZeroDivisionError: float division by zero

For now I will try to run one doc2vec at a time. But any input on this bug would be awesome.
Cheers.

**Comment by gojomo** [comment]
Thanks for the report. Duplicate of #518, let's discuss there instead.
